backend/models/purchases.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose')
+const { ObjectId } = mongoose.Types
+const purchaseSchema = new mongoose.purchase(
```

Comment:
The method name to create a mongoose Schema is `mongoose.Schema()`.
```suggestion
const purchaseSchema = new mongoose.Schema(
```